packages/augur-sdk/src/state/create-api.ts

```diff
@@ -208,6 +209,16 @@ export async function startServerFromClient(config: SDKConfiguration, client?: A
 export async function startServer(config: SDKConfiguration, account?: string): Promise<API> {
   const { api, sync } = await createServer(config, undefined, account);
 
+  api.augur.on(SubscriptionEventName.WarpSyncHashUpdated, async ({hash}) => {
```

Comment:
What if this is only registered when a config is set to do so AND there's a private key in the config?

Reply:
That seems reasonable.